src/components/TodoForm/styles.module.sass

```diff
@@ -1,9 +1,23 @@
 .input
     box-sizing: border-box
-    width: 100%
+    width: 85%
     padding: 20px 30px
     font-size: 2rem
     margin-bottom: 25px
     border: 1px solid #888
     outline: 0
 
+@mixin button()
```

Comment:
You shouldn't copy and paste the same code in multiple places. Instead you should have created a separate files for mixins